### ISSUE TITLE
"Copy to Clipboard" option when exporting image

### ISSUE THREAD

**Issue by GavHern** [photopea/photopea#3215]
I often find myself using photopea for quick/ minor edits or sketches that I just need to send over to a friend. It would be really helpful to be able to copy the image to my clipboard instead of finding a place to save it for a few seconds before I delete it. The option could fit pretty easily here:
![image](https://user-images.githubusercontent.com/47736824/117222234-dc38c980-adbf-11eb-9571-8cce279e82a1.png)

I apologize if there is already a way to do this, but at the very least, this would be a very intuitive place to put an option since I think this could be useful to a lot of people. Currently I go with screenshotting the image at as close to 100% zoom as possible, which obviously isn't a great solution

**Comment by MichalSZZ** [comment]
In your project:

1. You can use Ctrl+A for select everything (or with mouse in menu "Select"-> "All")
2. Shift+Ctrl+C for copy to clipboard everything inside that selection as a merged layers (or with mouse in menu "Edit"-> "Copy Merged")

**Comment by photopea** [comment]
Use Ctrl+A, and then, Shift+Ctrl+C  (Select All, and then, Copy Merged). 

Your communicator allows pasting images directly? Also, you can use File - Publish Online - Imgur (JPG), and get a link, which is more practical for sharing.

**Comment by GavHern** [comment]
Yes! You can paste emails into most things (discord, emails, twitter, etc.) and i would argue its more practical than a link to the image in most cases. I appreciate the tips for ctrl+shift+c, I will use it very frequently.
